### ISSUE TITLE
Add Pattern Matching to Wave.TypeLookupHandler so one could use wildcards in the type locations

### ISSUE THREAD

**Issue by erxdkh** [azist/azos#157]
TypeLookupHandler.cs:
```
 private Type lookupTargetInLocations(WorkContext work, string typeName)
{
  //need to look within namespaces
//as namespace may now have a *
 type-location { assembly="Erx.Business.Api.dll" ns{ name="Erx.Business.Api.Controllers.*" } }

}
```

**Comment by zhabis** [comment]
Addressed in  2022472  Unit tests are passing.

**Comment by zhabis** [comment]
Can now use something like:
```csharp
type-location 
{
   assembly="My1.dll" 
   ns{ name="My.Business.Api.Controllers.*" } 
   ns{ name="My.Business.Management.Controllers.*" } 
}

type-location 
{
   assembly="My2.dll" 
   ns{ name="My2.Web*Controllers*" } 
}
```
